tests/functional/test_venv.py

```diff
@@ -47,3 +48,22 @@ def test_venv_log_run_error2(tmp_path):
     echo = '/usr/bin/echo'
     with pytest.raises(ValueError, match=rf'^{echo!r} must not be an absolute path!'):
         runner.log_run([echo, "This also won't work!"])
+
+
+def test_fake_venv_python():
+    with mock.patch.object(
+        subprocess_util, 'async_log_run', wraps=subprocess_util.async_log_run
+    ) as log_run:
+        args = ['python', '-c', 'import antsibull_core; print("Hello, world!")']
+        p = FakeVenvRunner.log_run(args)
+        assert p.stdout == 'Hello, world!\n'
+
+        args[0] = sys.executable
```

Comment:
Does `sasert_called_once_with` make a deep copy of everything? If not, you should better copy this before modifying:
```suggestion
        args = args.copy()
        args[0] = sys.executable
```